File: program67.cpp
#include<iostream>
using namespace std;

class Pattern
{
    public:
        int iNo;

    Pattern(int A)
    {
        iNo = A;
    }
    int PatternDisplay()
    {   
        int iCnt = 0;
        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            cout<<" "<<iCnt;
        }
    }

};

int main()
{
    int iValue = 0;
    
    cout<<"Enter the number :"<<endl;
    cin>>iValue;

    Pattern obj(iValue);
    obj.PatternDisplay();

    return 0;
}

File: program67.java
import java.util.*;


class Pattern
{
    public int iNo;

    public Pattern(int A)
    {
        iNo = A;
    }
    public void DisplayPattern()
    {
        int iCnt = 0;
        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print(" "+iCnt);
        }
    }
}

class program67
{
    public static void main(String Arg[])
    {
        Scanner obj = new Scanner(System.in);
        int iValue = 0;

        System.out.println("Enter number:");
        iValue =obj.nextInt();

        Pattern pobj = new Pattern(iValue);
        pobj.DisplayPattern();


    }
}